docs/gallery/interactive/pyfar_filtering.ipynb

```diff
@@ -0,0 +1,281 @@
+    "\n",
+    "ax = pf.plot.time_freq(output, unit='samples')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Filtering with multi-channel Filters\n",
+    "\n",
+    "pyfar also contains filter banks, which can be seen as a collection of filters or a multi-channel filter. One example is the [Gammatone filter bank](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.GammatoneBands), which is implemented as a filter class (in the future all filter banks in pyfar will be classes). If we apply a multi-channel filter, the output becomes multi-channel as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the filter bank\n",
+    "Gammatones = pf.dsp.filter.GammatoneBands(freq_range=[20, 20_000])\n",
+    "\n",
+    "# Apply the filter bank to an impulse with a duration of 4096 samples\n",
+    "impulse_filter_bank, _ = Gammatones.process(pf.signals.impulse(4096))\n",
+    "ax = pf.plot.freq(impulse_filter_bank)\n",
+    "ax.set_ylim(-60, 5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Block-wise filtering\n",
+    "\n",
+    "For the above, output will be the same no matter how often `process` is called. This default behavior is often desired. In some cases, like block-wise filtering of a signal, the filter object can track the state of the filter. By default, the filter state is set to `None`, but a different state can be set at any time. This requires the `cshape` of the signal that will be filtered and the kind of state. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "comb_filter.init_state(input.cshape, 'zeros')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the same result as above can be obtained in blockwise processing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "block_1 = comb_filter.process(input[:, :128])  # compute first 128 samples\n",
+    "block_2 = comb_filter.process(input[:, 128:])  # compute last 128 samples\n",
+    "\n",
+    "# plotting the time signals of both blocks\n",
+    "ax = pf.plot.time(block_1, unit='samples', label='block 1')\n",
+    "pf.plot.time(pf.TimeData(block_2.time, np.arange(128)+128), label='block 2')\n",
+    "ax.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Time-variant and block-wise filtering\n",
+    "\n",
+    "If tracking the state of a filter object is enabled as in the example above, a time-variant filter can be realized by changing the filter coefficients between calls of the `process` function. Note that this however, only works if the order of the filter does not change."
```

Comment:
maybe add a link to ``process`` function

Reply:
I would not do it here, since each filter class has a process function and we did not link in the previous examples. Instead, I added links to the filter classes (that contain the docs for process) in the bullet point list above.